client/CBitmapHandler.cpp

```diff
@@ -109,43 +112,43 @@ SDL_Surface * BitmapHandler::loadBitmapFromDir(std::string path, std::string fna
 		logGlobal->warnStream() << "Call to loadBitmap with void fname!";
 		return nullptr;
 	}
-	if (!CResourceHandler::get()->existsResource(ResourceID(path + fname, EResType::IMAGE)))
+	if(!CResourceHandler::get()->existsResource(ResourceID(path + fname, EResType::IMAGE)))
 	{
 		return nullptr;
 	}
 
-	SDL_Surface * ret=nullptr;
+	SDL_Surface * ret = nullptr;
 
 	auto readFile = CResourceHandler::get()->load(ResourceID(path + fname, EResType::IMAGE))->readAll();
 
-	if (isPCX(readFile.first.get()))
-	{//H3-style PCX
+	if(isPCX(readFile.first.get()))
+	{ //H3-style PCX

```

Comment:
Opening brace is a perfect place for a comment - it carries no other message, but lets us save some vertical space.